src/test/java/com/coherent/task40/AlertsTest.java

```diff
@@ -0,0 +1,58 @@
+package com.coherent.task40;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import static com.coherent.task40.TestConstants.*;
+
+public class AlertsTest {
+    WebDriver driver = new ChromeDriver();
+    By confirmBoxButton = By.xpath("//button[@onclick='myConfirmFunction()']");
+    By alertBoxButton = By.xpath("//button[@onclick='myAlertFunction()']");
+    String confirmBoxText = "Press a button!";
+    String alertBoxText = "I am an alert box!";
```

Comment:
I don't see the need to create variable on the class level. As it used only in one test and haven't reused in other tests, the variable can be created on test(method) level. Or you can use the value in verification without variable creation( as you did in test JavaScriptConfirmBoxTest2())